desktop-app/src/pages/SimulationInsights.jsx

```
@@ -0,0 +1,450 @@
+  const avgScore = learnings.length > 0
+    ? learnings.reduce((sum, l) => sum + (l.confidence || 0), 0) / learnings.length
+    : 0;
+
+  return (
+    <div className="p-6 space-y-6">
+      {/* Header */}
+      <div>
+        <button
+          onClick={() => navigate(`/simulations/${id}`)}
+          className="flex items-center text-sm text-gray-600 hover:text-gray-900 mb-4"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)